assets_src/config/strings/en-GB.yaml

```diff
@@ -50,4 +50,39 @@ en-GB:
+    codex_rules_building_title: Building Structures
+    codex_rules_building_text: "> Villagers can build any structure in the game, provided they have the gold and the tech level to do so.\n> Some structures require multiple actions to build. You can do this with many villagers over one turn or one villager over multiple turns."
+
+    codex_rules_pop_cap: Population
+    codex_rules_pop_cap_title: Population Capacity
+    codex_rules_pop_cap_text: "> Every player has a set max on the number of units they can recruit. This is called your 'Population Cap.' If your army exceeds this number, you cannot recruit more units.\n> Your Population Cap can be increased by building Villages and extra Strongholds.\n- Each village adds 4 units to your popluation cap.\n- Each Stronghold adds 8 units.\n> You can check your current army size and Population Cap by inspecting your Stronghold. The number under the villager is your Population Cap, and the number above the villager is your current army size.\n> The highest possible Population Cap is 100"
+
+    codex_rules_artifacts: Artifacts
+    codex_rules_artifacts_title: Artifacts
+    codex_rules_artifacts_text: "> Artifacts are items and weapons that provide attack bonuses and special actions for Commanders, giving an edge in head-to-head combat.\n> Only Commanders can use artifacts, but almost any attack can destroy one.\n> Each Commander can use up to 6 artifacts, and can put down artifacts whenever necessary.\n- Daggers add 3% to the minimum damage of an attack\n- Axes add 3% to maximum damage of an attack\n- Sword add 2% to overall damage\n- Enchanted Blades add 2% overall damage, 3% minimal damage and 3% maximal damage\n- Armor reduces minimal damage of incoming attacks by 3%\n- Helmets reduce maximum damage of incoming attacks by 3%\n- Shields reduce overall damage of incoming attacks by 2%\n- Magical Shields reduce overall damage by 2%, minimal damage by 3% and maximum damage by 3%\n- Dimensional Doors allow Commanders to teleport to an empty space by an allied Stronghold. A Dimensional Door can be used once every 5 turns\n- Health Potions (red) recover health by 25%\n- Groove Potions (blue) recover groove charge. The amount gained depends on the groove's charge speed\n- Bows allow Commanders to attack up to 3 tiles away and target water and flying units. Bow attacks deal half of the normal damage."
+
+    codex_rules_shop: Shops
+    codex_rules_shop_title: Buying Artifacts
+    codex_rules_shop_text: "> Artifacts can be bought by Commanders (and only Commanders) at shops.\n> Shops sell five random artifacts and restock every 5 turns"
+
+    codex_rules_repair: Repairing
+    codex_rules_repair_title: Repairing Structures
+    codex_rules_repair_text: "> Villagers can repair structures to improve their sustainability.\n> Repairing costs 10 gold and restores up to 20% health.\n> Because structures no longer regenerate health each turn, repairing is the only way for structures to regain health."
```

Comment:
repairing costs half of building's original cost * the amount of health repaired